index.html

```diff
@@ -178,8 +178,13 @@ <h3>Stream mapping</h3>
         <li><code>-map 0:0 -map 0:2</code> means ‘take the first and third streams from the first input file’.</li>
         <li><code>-map 0:1 -map 1:0</code> means ‘take the second stream from the first input file and the first stream from the second input file’.</li>
       </ul>
-      <p>To map <em>all</em> streams in the input file to the output file, use <code>-map 0</code>. However, note that not all container formats can include all stream types: for example, .mp4 cannot contain timecode.</p>
       <p>When no mapping is specified in an ffmpeg command, the default for video files is to take just one video and one audio stream for the output: other stream types, such as timecode or subtitles, will not be copied to the output file by default. If multiple video or audio streams are present, the best quality one is automatically selected by FFmpeg.</p>
+      <p>To map <em>all</em> streams in the input file to the output file, use <code>-map 0</code>. However, note that not all container formats can include all stream types: for example, .mp4 cannot contain timecode.</p>
+
```

Comment:
I guess, the empty line inside the recipe is not needed.

Reply:
Now fixed in 10341e1 - cheers :-)